test/chemistry/test_oovqe.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+""" Test OOVQE """
+
+import unittest
+from test.aqua import QiskitAquaTestCase
+from ddt import ddt
+from qiskit.aqua import aqua_globals
+from qiskit.aqua import QuantumInstance
+from qiskit.aqua.components.optimizers import COBYLA
+from qiskit.chemistry.core import Hamiltonian
+from qiskit.chemistry.drivers import PySCFDriver, UnitsType
+from qiskit.chemistry.components.variational_forms import UCCSD
+from qiskit.chemistry.components.initial_states import HartreeFock
+from qiskit.chemistry.core import TransformationType, QubitMappingType
+from qiskit.chemistry.algorithms.minimum_eigen_solvers import OOVQE
+from qiskit.aqua.operators.expectations import MatrixExpectation
+from qiskit import Aer
+
+
+@ddt
+class TestOOVQE(QiskitAquaTestCase):
+    """ Test of the OOVQE algorithm"""
+
+    def setUp(self):
+        super().setUp()
+        self.energy_vqe = -2.77  # energy of the VQE with pUCCD ansatz and LBFGSB optimizer
+        self.initial_point = [0.039374, -0.47225463, -0.61891996, 0.02598386, 0.79045546,
+                              -0.04134567, 0.04944946, -0.02971617, -0.00374005, 0.77542149]
+        self.seed = 50
+        aqua_globals.random_seed = self.seed
+        driver = PySCFDriver(
```

Comment:
PySCF is not supported on all platforms. Maybe save an HDF5 file and use the HDf5 driver so these tests will work on any platform. Another not quite as good solution is to make the test skip if PySCF is not installed - like is done elsewhere.

Reply:
Yes, Anton seems to have problems running it. This might solve that issue. I think on Windows there is a high chance of having trouble.

Reply:
I confirm, now the test is ok on Linux and Windows.